src/style.css

```diff
@@ -1 +1,119 @@
-/* add styles here */
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+@font-face {
+  font-family: Avenir-bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+body {
+  font-family: Avenir, Helvetica, sans-serif;
+  font-size: 16px;
+  font-weight: 300;
+
+  --color-text: #3d4e61;
+  --color-bg: #fff;
+}
+
+.big__form {
+  position: relative;
+  margin-top: 20px;
+  display: block;
+}
+
+.icon {
+  display: block;
+  position: absolute;
+}
+
+.icon--big {
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
+}
+
+.input--big {
+  height: 70px;
+  margin-bottom: 20px;
+  padding-left: 62px;
+}
+
+.input {
+  font-size: 16px;
+  font-family: inherit;
+  background-color: var(--color-bg);
+  border: 1px solid rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  width: 100%;
+  font-weight: 300;
+}
+
+.icon--small {
+  width: 11px;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+}
+
+.small__form {
+  position: relative;
+  display: block;
+}
+
+.input--small {
+  height: 42px;
+  padding-left: 33px;
+}
+
+.input--big::placeholder {
+  font-weight: 300;
+  color: var(--color-text);
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.input--small::placeholder {
+  font-style: normal;
+  font-weight: 300;
+  font-size: 14px;
+  line-height: 19px;
+  color: var(--color-text);
+}
+
+.input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.input--big:focus {
+  border-radius: 3px;
+  color: var(--color-text);
+  font-size: 16px;
+  line-height: 22px;
+  text-shadow: (0 4px 4px rgba(0, 0, 0, 0.25));
+  font-weight: 800;
+  background: linear-gradient(180deg, var(--color-bg), 0%, #f6f6f7, 100%);
+  outline: none;
+}
+
+.input--small:focus {
+  border-radius: 3px;
+  color: var(--color-text);
+  font-size: 14px;
+  line-height: 19px;
+  font-weight: 800;
+  background: linear-gradient(180deg, var(--color-bg), 0%, #f6f6f7, 100%);
+  outline: none;
+}
```

Comment:
create some another class and move all common styles there, it will simplify your code